packages/fluent/scss/datetimepicker/_layout.scss

```diff
@@ -0,0 +1,56 @@
+@use "../core/" as *;
+@use "_variables.scss" as *;
+@use "../action-buttons/_variables.scss" as *;
+
+@mixin kendo-datetimepicker--layout() {
+
+    // Datetime container
+    .k-datetime-container {}
+
+    // Datetime selector
+    .k-datetime-selector {
+        display: flex;
+        transition: transform .2s;
+    }
+
+    // Wrap
+    .k-datetime-wrap {
+        width: var( --kendo-datetime-width, #{$kendo-datetime-width} );
+        overflow: hidden;
+
+        .k-datetime-buttongroup {
+            padding-inline: var( --kendo-datetime-button-group-padding-x, #{$kendo-actions-padding-x} );
+            padding-block: var( --kendo-datetime-button-group-padding-y, #{$kendo-actions-padding-y} )
+        }
+
+        .k-calendar,
+        .k-timeselector {
+            width: 100%;
+            border-width: 0;
+        }
+    }
+
+    // Inner wrap
+    .k-datetime-calendar-wrap,
+    .k-datetime-time-wrap {
+        text-align: center;
+        flex: 0 0 var( --kendo-datetime-width, #{$kendo-datetime-width} );
+    }
+
+    // Tabs
+    .k-date-tab .k-datetime-selector {
+        transform: translateX(0);
+    }
+    .k-time-tab .k-datetime-selector {
+        transform: translateX(-100%);
+    }
+
+    k-datetimepicker {
+        &[dir="rtl"],
+        .k-rtl & {
+            .k-time-tab .k-datetime-selector {
+                transform: translateX(100%);
+            }
+        }
+    }
+}
```

Comment:
the datetime selector does not look okay in the pictures, I'll take a look at it tmrw

Reply:
in an ad-hoc discussion we agreed to resolve this with a separate PR for the Calendar